test/PostgreSQL/Test.hs

```diff
@@ -330,3 +554,11 @@ run_worker act = withConn $ runSqlConn (migrateIt >> act)
 withConn :: RunDbMonad m => (SqlBackend -> R.ResourceT m a) -> m a
 withConn =
   R.runResourceT . withPostgresqlConn "host=localhost port=5432 user=esqutest password=esqutest dbname=esqutest"
+
+-- | Show the SQL generated by a query
+showQuery :: (Monad m, ES.SqlSelect a r, BackendCompatible SqlBackend backend)
```

Comment:
This is pretty handy, I've wanted a glued-together `rawSql/toRawSql` like this before. Hopefully people are smart and don't try to use this for anything other than logging/debugging.

Reply:
This is only in the test module and not exported.